### ISSUE TITLE
MAPI_E_LOGON_FAILURE

### ISSUE THREAD

**Issue by tedlwsn** [jankeirse/tvhgooglemapi#37]
Sometimes receive the above error when trying to send email. Requires repair to fix, but would like a permanent solution, if possible.  We have many people in our office using this, and don't want to have to run repair for someone each week.
![image](https://cloud.githubusercontent.com/assets/2120219/7596339/8829c10a-f8b2-11e4-8584-7472d76acf18.png)

**Comment by jankeirse** [comment]
Interesting, I've never seen this, what is the repair you use to fix?
Could you, in dos, in the installation folder, on a machine that gives this error, execute gmaildrafter and give the output? 
Like this:
c:\>cd "C:\Program Files (x86)\tvhgooglemapi"
C:\Program Files (x86)\tvhgooglemapi>java -jar gmaildrafter.jar

**Comment by tedlwsn** [comment]
The repair that fixed the issue was just going into Control Panel then Programs & Features like you were going to uninstall it and then Clicking Repair instead of Uninstall. It has occurred on 2 machines so far and the doing the repair seems to fix it. Once it occurs again I'll run the command you posted and let you know what happens.

**Comment by tedlwsn** [comment]
We got the error again a few times and figured out what the issue is after some testing. It does that whenever a user tries to send a file that is larger than 25MB, which of course GMail maxes out at 25MB for attachments so it makes sense why it trips out. However in order to get the GMAPI to work again I have to run the repair and restart the computer. Because when that error hits usually the GMAPI stops working.

I ran the command you recommended and it worked like it should and pulled up an empty draft in Gmail, however until I run the repair and restart the user's PC the GMAPI won't work when a program calls it after the error occurs. The specific program that throws the error like that is LaserFiche. You get a similar error in Word/Excel but the prompt is different.

If there was an exception that stopped the MAPI program from starting if the file size is over 25MB that might help stop the issue.
